include/nds/dma.h

```diff
@@ -66,18 +66,20 @@ extern "C" {
 #define DMA_BUSY            BIT(31)
 #define DMA_IRQ_REQ         BIT(30)
 
-#define DMA_START_NOW       0
-#define DMA_START_CARD      (5 << 27)
+#define DMA_START_NOW       0         ///< Start DMA immediately
 
 #ifdef ARM7
-#    define DMA_START_VBL   BIT(27)
+#    define DMA_START_VBL   (1 << 28) ///< Auto-start DMA on vertical blank
+#    define DMA_START_CARD  (2 << 28) ///< Auto-start DMA on Slot-1 card transfer
 #endif
```

Comment:
Can you add defines for option 3?

> 3  DMA0/DMA2: Wireless interrupt, DMA1/DMA3: GBA Cartridge Slot

It could be nice so that we use it in DSWiFi or a successor.

Reply:
Do we still want to use `CARD`/`CART` nomenclature going forward, or migrate towards `SLOT1`/`SLOT2`?

Reply:
I think `SLOT2` is way more clear, no question about that. 😅

Reply:
Should be fixed now.